README.md

```diff
@@ -27,3 +27,8 @@ Start the service:
 Then open the page with your favorite browser:
 
     http://localhost:8080/validator.html
+
+
+<!-- Badges -->
+[travis-ci]: https://travis-ci.org/spacedirectory/validator
+[travis-ci-badge]: https://img.shields.io/travis/spacedirectory/validator.svg
```

Comment:
Shouldn't you specify the branch? Like `https://travis-ci.org/spacedirectory/validator.svg?branch=master`